third_party/sfml.BUILD

```diff
@@ -54,6 +61,10 @@ cc_library(
             "src/SFML/Window/Unix/*.cpp",
             "src/SFML/Window/Unix/*.hpp",
         ]),
+        "@platforms//os:macos": glob([
+            "src/SFML/Window/Unix/*.cpp",
+            "src/SFML/Window/Unix/*.hpp",
+        ]),
```

Comment:
SFML has macOS-specific implementations of most things: https://github.com/SFML/SFML/tree/ea42079da9b72f81d49bfea476e7c8cd3fff97a0/src/SFML/Window/macOS

If you don't want to deal with that right now, feel free to drop the macos-fixes here and just mark the library as incompatible together with a TODO for someone to deal with later.

Just having most of our things building on macOS in CI is a massive improvement, so I'm more than happy to push some things like this to the future.

Reply:
A lot of the macOS-specific code is written in objective-C or objective-C++, and it doesn't look like you can include those sources in a `cc_library` rule. Probably have to use `objc_library` instead, so I'm not sure what the nicest way to do this is.

Reply:
I *think* a objc_library that sfml:window links when platforms/os:macos would work, but tbh I've only touched that part of Bazel when dealing with Android. Probably just mark it as incompatible for now since that could turn into a pretty large chunk of work if you're unlucky.